tendrl/commons/objects/definition/master.yaml

```diff
@@ -83,11 +83,14 @@ namespace.tendrl:
 
   objects:
     Cluster:
-      internal: True
       enabled: True
       help: "Represents a cluster"
       list: /clusters
+      value: clusters/$TendrlContext.integration_id
       attrs:
+        integration_id:
+          help: integration id
+          type: String
```

Comment:
Confused `integration_id` with `node_identifier`. Changed the type of `node_identifier` as list.